config/settings.yml

```diff
@@ -1,6 +1,7 @@
-dor_services:
-  url: ~
-  token: ~
+sdr_api:
```

Comment:
And confirming here: we already have this in shared_configs for all three deployed environments.

Reply:
exactly.